pkg/cmd/cli/cmd/rollback.go

```diff
@@ -57,14 +60,14 @@ func NewCmdRollback(fullName string, f *clientcmd.Factory, out io.Writer) *cobra
 	}
 
 	cmd := &cobra.Command{
-		Use:     "rollback DEPLOYMENT",
+		Use:     "rollback [dc/]config-name",
```

Comment:
> Don't

@ironcladlou I'm not sure how you can distinguish the different cases (old arg versus new arg).  Also, the fact that `hack/test-cmd.sh` didn't require any changes suggests this needs to be added there.